x86/SearchMacros.asm

```diff
@@ -354,54 +354,62 @@ Display	2, "Search(alpha=%i1, beta=%i2,	depth=%i8, cutNode=%i9)	called%n"
 		cmp   esi, dword[.evalu]
 		 jg   .8skip
 		add   esi, 225
-    if USE_MATEFINDER =	0
-	      movzx   ecx, word[rbx+State.npMaterial+2*rcx]
```

Comment:
Lines 358-360 are no longer a part of this section. Residue from my patch https://github.com/lantonov/asmFish/commit/1859f0a60835c59e193d1f24c6638e6573ddb910

and later restoration of unintentional mateFinder deletions